### PR TITLE
Updated contract addresses and abis used for indexing

### PR DESCRIPTION
**Pull request by omadoyeabraham**
- Updated contract addresses used in subquery indexers
- Updated indexers to use new field names from updated contracts

### CODE REVIEW

**Review by seunlanlege — APPROVED** [review]

File: indexers/src/handlers/events/evmHost/postRequest.event.handler.ts [hunk 1]
```diff
@@ -32,14 +35,14 @@ export async function handlePostRequestEvent(
...
-    data,
+    body
   );
 
   // Create the request entity
   await RequestService.findOrCreate({
     chain,
     commitment: request_commitment,
-    data,
+    data: body,
```
Comment (seunlanlege) [inline indexers/src/handlers/events/evmHost/postRequest.event.handler.ts:45]: can we change this field name to `body`

**Comment by seunlanlege** [comment]
Also i think we should change all height fields from BigInt to normal Ints so they're not strings and we can actually make queries based on filters

File: indexers/schema.graphql [hunk 2]
```diff
@@ -6,189 +6,189 @@
...
 """
 enum SupportedChain {
-    ETHE
-    ARBI
-    OPTI
-    BASE
-    BSC
-    POLY
-    HYPERBRIDGE
+  ETHE
```
Comment (Wizdave97) [inline indexers/schema.graphql:9]: We've changed the state machine ids for ethereum chains, you need to update the`extractStateMachineIdFromSubstrateEventData`

Or is there a way to remove this enum and just use a string to represent state machines all through

File: indexers/src/addresses/state-machine.addresses.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      startBlock: 8906802,
+      isEvm: true,
+      isEvmL2: true,
+      ethereumHostAddress: "0x27D689e361ab92aCab04Ea21c1B3F507A94a9DAd",
+    }, // Optimism Sepolia
+    "EVM-421614": {
+      chainId: "421614",
+      startBlock: 20034995,
+      isEvm: true,
+      isEvmL2: false,
```
Comment (Wizdave97) [inline indexers/src/addresses/state-machine.addresses.ts:36]:
```suggestion
      isEvmL2: true,
```

File: indexers/src/addresses/state-machine.addresses.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      ethereumHostAddress: "0x15Ba7e42BC2c3e8FeDEb30D13CEE611D97315E7F",
+    }, // Arbitrum Sepolia
+    BSC: {
```
Comment (Wizdave97) [inline indexers/src/addresses/state-machine.addresses.ts:39]:
```suggestion
   "EVM-97": {
```

**Review by Wizdave97 — APPROVED** [review]

**Pull request merged**